cwltool/argparser.py

```diff
@@ -250,6 +250,12 @@ def arg_parser():  # type: () -> argparse.ArgumentParser
                         help="Enable loading and running cwltool extensions "
                              "to CWL spec.", default=False)
 
+    exgroup = parser.add_mutually_exclusive_group()
+    exgroup.add_argument("--enable-color", action="store_true",
+                        help="Enable colored logging (default true)", default=True)
+    exgroup.add_argument("--disable-color", action="store_false", dest="enable_color",
+                        help="Enable colored logging (default true)")
```

Comment:
??

Reply:
Are you referring to the cut and paste error in the help text?

Reply:
Yes